cmd/kstone-controller/etcdinspection-controller/controller.go

```diff
@@ -112,6 +117,14 @@ func (c *EtcdInspectionCommand) Run() error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	if c.enableProfiling {
+		go func() {
+			addr := "127.0.0.1:6060"
```

Comment:
It is inconvenient to troubleshoot if only listening 127.0.0.1.

Reply:
fixed.